Moq.Contrib.HttpClient.Test/RequestExtensionsTests.cs

```diff
@@ -248,10 +248,6 @@ public async Task MatchesQueryParameters()
             Action verifyThreeFoos = () => handler.VerifyRequest(fooUrl, Times.Exactly(3), "oh noes");
             Action verifyFooPosted = () => handler.VerifyRequest(HttpMethod.Post, fooUrl);
             Action verifyBarPosted = () => handler.VerifyRequest(HttpMethod.Post, barUrl, failMessage: "oh noes");
-            Action verifyFooPostedStuff = () => handler.VerifyRequest(HttpMethod.Post, fooUrl,
-                async r => (await r.Content.ReadAsStringAsync()) == "stuff");
-            Action verifyFooPostedOtherStuff = () => handler.VerifyRequest(HttpMethod.Post, fooUrl,
-                async r => (await r.Content.ReadAsStringAsync()) == "other stuff", failMessage: "oh noes");
```

Comment:
It seems in .NET Framework the HttpRequestMessage gets disposed, which prevents us from accessing the request body in Verify. That said, these overloads aren't especially useful anyway; they were generated alongside the SetupRequest methods, but the `match` predicate is meant for setups, so it's probably safe to leave them out (and avoid encouraging their use).